models/game_room.go

```diff
@@ -180,9 +181,12 @@ func createPod(
 			containers[i].Env = append(containers[i].Env, namesEnvVars...)
 		}
 
-		pod, err = NewPodWithContainers(name, containers, configYAML, clientset, redisClient, mr)
+		pod, err = NewPodWithContainers(name, containers, configYAML, redisClient, mr)
 	}
+
 	if err != nil {
+		// NOTE: We will get an error here if we cannot access redis. This implies that redis is probably not accessible,
+		// se we were not successful in adding the room's keys as well. That is why we don't call room.ClearAll here.
```

Comment:
I believe there is something missing or mistyped in the beginning of the second line of this note